docs/index.adoc

```diff
@@ -8,6 +8,8 @@ An image containing all you need to run a WildFly Server. This image is to be us
 
 * JDK17 based runtime image: `docker pull quay.io/wildfly/wildfly-runtime:latest-jdk17`
 
+* JDK21 based runtime image: `docker pull quay.io/wildfly/wildfly-runtime:latest-jdk21`
+
 * Latest LTS JDK based runtime image: `docker pull quay.io/wildfly/wildfly-runtime:latest`
```

Comment:
@jfdenise Should we reformulate to be consistent with [Brian's words for WildFly](https://www.wildfly.org/news/2023/10/18/WildFly30-Released/)?

Afaict, `latest` continues to point to `latest-jdk17`, not `latest-jdk21` right?

Reply:
Good point. Fixed.